src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -62,6 +66,31 @@ export default class DataTable extends Vue {
   _paginationListenersAdded = false;
   _chiMajorVersion = 5;
 
+  _toggleInfoPopover(infoPopoverRef: string) {
+    const popover = this.$refs[infoPopoverRef];
+
+    if (popover) {
+      // eslint-disable-next-line
+      // @ts-ignore
+      popover.toggle();
+    }
+  }
+
+  _getDescription(description: string | DataTableColumnDescription) {
+    const template = (description as DataTableColumnDescription).template;
+
+    if (template) {
+      const getDescriptionSlot = this.$scopedSlots[template];
```

Comment:
```suggestion
      const descriptionSlot = this.$scopedSlots[template];
```
Why get* ?

Reply:
because it's a function so i thought it was more readable that way but descriptionSlot works!